pkg/cmd/server/apis/config/validation/master.go

```diff
@@ -242,7 +242,7 @@ func ValidateAuditConfig(config configapi.AuditConfig, fldPath *field.Path) Vali
 		} else {
 			policyConfiguration, ok := config.PolicyConfiguration.(*auditinternal.Policy)
 			if !ok {
-				validationResults.AddErrors(field.Invalid(fldPath.Child("policyConfiguration"), config.PolicyConfiguration, "must be of type audit/v1alpha1.Policy"))
```

Comment:
message is wrong.  it should allow both.

Reply:
If you have audit/v1alpha1 it won't complain. If you don't I'd prefer you use v1beta1, that's why I went with that.